dds/DCPS/ContentFilteredTopicImpl.cpp

```diff
@@ -30,7 +30,14 @@ ContentFilteredTopicImpl::ContentFilteredTopicImpl(const char* name,
   , filter_eval_(filter_expression, false /*allowOrderBy*/)
   , expression_parameters_(expression_parameters)
   , related_topic_(DDS::Topic::_duplicate(related_topic))
-{}
+{
+  if (DCPS_debug_level > 5) {
+      ACE_DEBUG((LM_DEBUG,
```

Comment:
Please check formatting in terms of level of indent and the placement of () for a function call.